### DISCUSSION TITLE
All peer dependencies really needed?

### DISCUSSION THREAD

**Discussion by frontalicious-martijn** [JesusTheHun/storybook-addon-remix-react-router#57] [Q&A] [answered]
When I try to install this addon, I get this error:

```
Peer dependencies that should be installed:
@storybook/blocks@^7.0.0       
@storybook/components@^7.0.0   
@storybook/manager-api@^7.0.0  
@storybook/theming@^7.0.0      
@storybook/channels@^7.0.0    
@storybook/core-events@^7.0.0  
@storybook/preview-api@^7.0.0  
```

Are these (peer) dependencies really necessary to use this plugin? I rather not install all these since I don't use them.

**Comment by JesusTheHun** [comment, marked as answer]
Yes they are and they come with the addon essentials IIRC.
They are devDependencies, so they won't impact your runtime nor bundle, if that's what you are worried about.
